### ISSUE TITLE
NVDA does not work properly with the "Alarms" app in Windows 8.1

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3700]
**Reported by k_kolev1985 on 2013-12-08 16:27**
NVDA does not read correctly or at all some controls in the "Alarms" application on Windows 8.1.
1. In the dialog for creating a new or editing an alarm, NVDA does not read the minutes and hours values of the round sliders, with witch their set. Instead, it reads their percentage values, witch is not very helpful to define for what time is the alarm set at present.
2. Another problem occurs when we try in that same dialog to choose the alarm sound for the alarm. We open the drop-down menu with Alt+Down arrow, the drop-down menu opens, but NVDA does not read the text name of the alarm sound witch is either selected with the arrow keys or by pointing with the mouse (with "mouse tracking" enabled in NVDA).
3. In the drop-down menu for alarm sound selection, on the left of each sound, there is a "play" button, with witch we can preview the sound before choosing it. I can't activate that button with either spacebar or Enter.

Note: When an alarm sound is selected in the drop-down menu, if I use object navigation to go down a level, I can find and read the text label for the selected sound and the "preview" button and activate it to play the selected sound. All this - with object navigation, witch once again proves how useful it can be at times ;).
1. When the alert window for an occurring alarm appears on the screen, there is no way to get to it (at least as far as I know) to interact with it. But I guess this is rather an Windows itself issue, and not to nVDA.

But for the other ones: can something in NVDA be done so the work with the "Alarms" application is a little bit more easier?

Version of NVDA: 2013.3 (installed);
Version of Windows: Windows 8.1, 32-bit, with regional and
locale settings set to "Bulgarian".

**Comment by Adriani90** [comment]
@k-kolev1985 is this still an issue?

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, it is. I can still reproduce it with the latest NVDA. I've tested it just now in my virtual machine with Windows 8.1.

**Comment by Adriani90** [comment]
Support for Windows 8.1 has completely ended in January 2023, and most windows 8.1 apps are discontinued. I am closing this as abandoned, since I don't expect any further accessibility updates for this version.
If you are having the same issue on Windows 10 or newer, please open a new one with updated steps to reproduce.
